util/template/template.go

```diff
@@ -0,0 +1,44 @@
+package template
+
+import (
+	"bytes"
+	"io"
+
+	"github.com/valyala/fasttemplate"
+)
+
+const (
+	prefix = "{{"
+	suffix = "}}"
+)
+
+type Template interface {
+	Replace(replaceMap map[string]string, allowUnresolved bool) (string, error)
+}
+
+func New(s string) (Template, error) {
```

Comment:
```suggestion
func NewTemplate(s string) (Template, error) {
```
??? When used within this package it's not prefixed by `template.New` so it's very unclear what this does